src/ResultsParser.ts

```diff
@@ -211,7 +211,23 @@ export default class ResultsParser {
 
   updateResults(data: { testSuite: any }) {
     if (data.testSuite.tests.length > 0) {
-      this.result.push(data);
+      const resIndex = this.result.findIndex(
+        (res) => res.testSuite.title === data.testSuite.title,
+      );
+      if (resIndex > -1) {
+        for (const test of data.testSuite.tests) {
+          const testIndex = this.result[resIndex].testSuite.tests.findIndex(
+            (tes) => tes.name === test.name,
```

Comment:
This also excludes tests to be added per project that it fails - is this intentional? I would like it to report per project that fails a particular test.

Reply:
yea this was an oversight

Reply:
hey @willbertSmillbert are you good to fix this?

Reply:
@ryanrosello-og  the project thing appeared resolved by #171  I will have a look at the attachment issue tomorrow